tests/foreman/cli/test_contentviews.py

```diff
@@ -142,6 +142,46 @@ def setUp(self):
                 {u'organization-id': TestContentView.org['id']},
                 cached=True)
 
+    def test_bz1184442(self):
+        """@Test: Delete content view and verify it was actually deleted from
+        hard drive.
+
+        @Feature: Content View
+
+        @Assert: Content view was deleted and pulp folder doesn't contain
+        content view files anymore
+
+        @BZ: 1184442
+
+        """
+        # Create a content view and publish it
+        content_view = make_content_view({'organization-id': self.org['id']})
+        ContentView.publish({u'id': content_view['id']})
+        content_view = ContentView.info({u'id': content_view['id']})
+        # Check content view files presence before deletion
+        result = ssh.command(
+            'find /var/lib/pulp -name "*{0}*"'.format(content_view['name']))
+        self.assertEqual(result.return_code, 0)
+        self.assertNotEqual(len(result.stdout), 0)
+        # Completely delete the content view
+        self.assertEqual(len(content_view['versions']), 1)
+        ContentView.remove_from_environment({
+            'id': content_view['id'],
+            'lifecycle-environment-id':
+                content_view['lifecycle-environments'][0]['id'],
+        })
+        ContentView.version_delete({
+            'content-view': content_view['name'],
+            'organization': self.org['name'],
+            'version': content_view['versions'][0]['version'],
+        })
+        ContentView.delete({'id': content_view['id']})
+        # Check content view files presence after deletion
+        result = ssh.command(
+            'find /var/lib/pulp -name "*{0}*"'.format(content_view['name']))
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stdout), 0)
```

Comment:
Not the same. Before deletion it's `self.assertNotEqual`, after - `self.assertEqual`. 
It's bad idea to check the disk space as numerous tasks may run in background and hard disk space may be consumed by something else. Locally 1 of 10 runs was failing because of this.
Then I came up with another idea - each published content view stores some files in `/var/lib/pulp`, and all the related files can be easily found by content view name in the filenames. So I'm verifyng that content view files are actually present before deletion, and they actually disappear after deletion. IMO it's even better to check this way, as we're 100% sure that files of the specific content view were deleted.